api/pkg/apis/v1alpha1/utils/symphony-api.go

```diff
@@ -99,7 +99,7 @@ func getApiClient() (*apiClient, error) {
 	if ShouldUseSATokens() {
 		clientOptions = append(clientOptions, WithServiceAccountToken())
 	} else {
-		clientOptions = append(clientOptions, WithUserPassword(context.TODO(), "", ""))
+		clientOptions = append(clientOptions, WithUserPassword(context.TODO(), "admin", ""))
```

Comment:
I think it would better if we force the user of the ApiClient to supply a user name and password in this case instead of hard-code it.